cryoet_data_portal_neuroglancer/models/json_generator.py

```diff
@@ -116,13 +116,16 @@ class ImageJSONGenerator(RenderingJSONGenerator):
     volume_rendering_is_visible: bool = False
     volume_rendering_gain: float = 0.0
     can_hide_high_values_in_neuroglancer: bool = False
+    blend: str = "additive"
```

Comment:
Looks like the changes from https://github.com/chanzuckerberg/cryoet-data-portal-neuroglancer/pull/27 have made it to this PR. 😅